packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/StakeModal/StakeEthForm/Inputs.tsx

```diff
@@ -74,55 +82,102 @@ export const Inputs = () => {
 
     const displaySymbol = getNetworkDisplaySymbol(account.symbol);
 
+    const isFractionButtonDisabled = (divisor: number) => {
+        if (!account.formattedBalance || !network.decimals) return false;
+
+        return new BigNumber(account.formattedBalance)
+            .dividedBy(divisor)
+            .decimalPlaces(network.decimals)
+            .lte(MIN_ETH_AMOUNT_FOR_STAKING);
+    };
+
+    const tooltip = (
+        <Translation
+            id="TR_STAKE_MIN_AMOUNT_TOOLTIP"
+            values={{
+                amount: MIN_ETH_AMOUNT_FOR_STAKING.toString(),
+                symbol: displaySymbol,
+            }}
```

Comment:
merge conflicts not resolved correctly, symbol -> networkDisplaySymbol
![Screenshot 2025-01-07 at 8 28 54](https://github.com/user-attachments/assets/9fa9cf89-3352-449d-9d1f-23112da3bd53)